tests/tracer/test_span.py

```diff
@@ -155,9 +156,35 @@ def test_finish_no_tracer(self):
         s = Span(tracer=None, name="test.span")
         s.finish()
 
+    def test_init_span_with_tracer(self):
+        with warnings.catch_warnings(record=True) as ws:
+            warnings.simplefilter("always")
+
+            # Initialize Span with tracer
+            Span(self.tracer, "test_span")
+            assert len(ws) == 1
+            assert (
+                str(ws[0].message)
+                == "'ddtrace.Span.tracer' is deprecated and will be remove in future versions (1.0.0)."
+                + " Use Span(tracer=None, name, ...) instead."
```

Comment:
```suggestion
                == (
                    "'ddtrace.Span.tracer' is deprecated and will be remove in future versions (1.0.0)."
                    " Use Span(tracer=None, name, ...) instead."
                )
```


This is an alternative approach as well. Python will auto-concatenate sequential strings without the need for `+`, but they need to be apart of the same statement.

(not asking you to make this change, just sharing)

Reply:
That's a lot cleaner. I had to fix a test so I just added this change to my most recent push